TimeZone error when running GSF server from eclipse

When attempting to run the  Hello World app on the local server from inside Eclipse:

SEVERE: Failed to start server
java.lang.RuntimeException: Unable to restore the previous TimeZone
	at com.google.appengine.tools.development.devappserver2.DevAppServer2Impl.restoreLocalTimeZone(DevAppServer2Impl.java:341)
	at com.google.appengine.tools.development.devappserver2.DevAppServer2Impl.doStart(DevAppServer2Impl.java:244)
	at com.google.appengine.tools.development.devappserver2.DevAppServer2Impl.access$000(DevAppServer2Impl.java:56)
	at com.google.appengine.tools.development.devappserver2.DevAppServer2Impl$1.run(DevAppServer2Impl.java:208)
	at com.google.appengine.tools.development.devappserver2.DevAppServer2Impl$1.run(DevAppServer2Impl.java:206)
	at java.security.AccessController.doPrivileged(Native Method)
	at com.google.appengine.tools.development.devappserver2.DevAppServer2Impl.start(DevAppServer2Impl.java:206)
	at com.google.appengine.tools.development.devappserver2.StandaloneInstance$StartAction.apply(StandaloneInstance.java:128)
	at com.google.appengine.tools.development.devappserver2.StandaloneInstance$StartAction.apply(StandaloneInstance.java:82)
	at com.google.appengine.tools.util.Parser$ParseResult.applyArgs(Parser.java:48)
	at com.google.appengine.tools.development.devappserver2.StandaloneInstance.run(StandaloneInstance.java:66)
	at com.google.appengine.tools.development.devappserver2.StandaloneInstance.main(StandaloneInstance.java:44)
Caused by: java.lang.NoSuchMethodException: java.util.TimeZone.setDefaultInAppContext(java.util.TimeZone)
	at java.lang.Class.getDeclaredMethod(Class.java:2130)
	at com.google.appengine.tools.development.devappserver2.DevAppServer2Impl.restoreLocalTimeZone(DevAppServer2Impl.java:337)
	... 11 more

INFO     2016-05-18 18:05:07,230 module.py:788] default: "GET / HTTP/1.1" 500 2170
ERROR    2016-05-18 18:05:09,618 http_runtime.py:396] bad runtime process port ['']
May 18, 2016 6:05:08 PM com.google.appengine.tools.development.AbstractContainerService configure
WARNING: Null value for containerConfigProperties.get(devappserver.portMappingProvider)
May 18, 2016 6:05:08 PM com.google.apphosting.utils.jetty.JettyLogger info
INFO: Logging to JettyLogger(null) via com.google.apphosting.utils.jetty.JettyLogger
May 18, 2016 6:05:08 PM com.google.appengine.tools.development.devappserver2.StandaloneInstance$StartAction apply
ERROR    2016-05-18 18:05:09,623 http_proxy.py:116] bad runtime process port ['']
SEVERE: Failed to start server
java.lang.RuntimeException: Unable to restore the previous TimeZone
	at com.google.appengine.tools.development.devappserver2.DevAppServer2Impl.restoreLocalTimeZone(DevAppServer2Impl.java:341)
	at com.google.appengine.tools.development.devappserver2.DevAppServer2Impl.doStart(DevAppServer2Impl.java:244)
	at com.google.appengine.tools.development.devappserver2.DevAppServer2Impl.access$000(DevAppServer2Impl.java:56)
	at com.google.appengine.tools.development.devappserver2.DevAppServer2Impl$1.run(DevAppServer2Impl.java:208)
	at com.google.appengine.tools.development.devappserver2.DevAppServer2Impl$1.run(DevAppServer2Impl.java:206)
	at java.security.AccessController.doPrivileged(Native Method)
	at com.google.appengine.tools.development.devappserver2.DevAppServer2Impl.start(DevAppServer2Impl.java:206)
	at com.google.appengine.tools.development.devappserver2.StandaloneInstance$StartAction.apply(StandaloneInstance.java:128)
	at com.google.appengine.tools.development.devappserver2.StandaloneInstance$StartAction.apply(StandaloneInstance.java:82)
	at com.google.appengine.tools.util.Parser$ParseResult.applyArgs(Parser.java:48)
	at com.google.appengine.tools.development.devappserver2.StandaloneInstance.run(StandaloneInstance.java:66)
	at com.google.appengine.tools.development.devappserver2.StandaloneInstance.main(StandaloneInstance.java:44)
Caused by: java.lang.NoSuchMethodException: java.util.TimeZone.setDefaultInAppContext(java.util.TimeZone)
	at java.lang.Class.getDeclaredMethod(Class.java:2130)
	at com.google.appengine.tools.development.devappserver2.DevAppServer2Impl.restoreLocalTimeZone(DevAppServer2Impl.java:337)
	... 11 more

INFO     2016-05-18 18:05:09,728 module.py:788] default: "GET /_ah/warmup HTTP/1.1" 500 2170
ERROR    2016-05-18 18:05:09,873 http_proxy.py:116] bad runtime process port ['']
INFO     2016-05-18 18:05:09,975 module.py:788] default: "GET /demo/ HTTP/1.1" 500 2170


Possibly relevant:

https://github.com/GoogleCloudPlatform/gcloud-maven-plugin/issues/54
http://stackoverflow.com/questions/24310654/timezone-error-when-running-gwt-gae-app

--- a/plugins/com.google.cloud.tools.eclipse.appengine.localserver/gcloud_serverdef.xml
+++ b/plugins/com.google.cloud.tools.eclipse.appengine.localserver/gcloud_serverdef.xml
@@ -60,7 +60,7 @@
   </project>
 
   <start>
-    <external>"${cloudSdkDirectory}/bin/dev_appserver.py" --port ${port} "${webappDirectory}"</external>
+    <external>"${cloudSdkDirectory}/bin/dev_appserver.py" --port ${port} --jvm_flag=-Dappengine.user.timezone=UTC  "${webappDirectory}"</external>
    <!-- TODO we may not need this; check -->
     <workingDirectory>${serverRootDirectory}</workingDirectory>
     <debugPort>${debugPort}</debugPort>

